zap.conf

```diff
@@ -0,0 +1,121 @@
+# zap-full-scan rule configuration file
```

Comment:
are these suggestions from somewhere (owasp?), and is there customization we've done that would be helpful to know about?

Reply:
Um, I grabbed them from the complaint tracker repo I worked on with OneOHS. I honestly don't remember how we ended up with this exact set of warn/fails